clar2wasm/tests/standard/property_tests.rs

```diff
@@ -504,3 +504,165 @@ fn prop_log2_int() {
         }
     })
 }
+
+#[test]
+fn prop_bit_and() {
```

Comment:
Why no tests for signed ints? (and same question for most following tests)

Reply:
Added signed/unsigned test for all bitwise ops